utils/evaluate_utils.py

```diff
@@ -92,9 +93,50 @@ def class_PR_curves(
     return fig, PR_data
 
 
+def model_cm(
+    log_reg_model: LogisticRegression, dataset: pd.DataFrame
+) -> pd.DataFrame:
+    """
+    display confusion matrix for logistic regression model on dataset
+
+    Parameters
+    ----------
+    log_reg_model : LogisticRegression
+        logistic regression model to evaluate
+    dataset : pd.DataFrame
+        dataset to evaluate model on
+
+    Returns
+    -------
+    pd.DataFrame
+        confusion matrix of model evaluated on dataset
+    """
+
+    # get features and labels dataframes
+    X, y = get_X_y_data(dataset)
+
+    # get predictions from model
+    y_pred = log_reg_model.predict(X)
+
+    # create confusion matrix
+    conf_mat = confusion_matrix(y, y_pred, labels=log_reg_model.classes_)
+    conf_mat = pd.DataFrame(conf_mat)
+    conf_mat.columns = log_reg_model.classes_
+    conf_mat.index = log_reg_model.classes_
+
+    # display confusion matrix
+    plt.figure(figsize=(15, 15))
+    ax = sns.heatmap(data=conf_mat, annot=True, fmt=".0f", cmap="viridis", square=True)
+    ax = plt.xlabel("Predicted Label")
+    ax = plt.ylabel("True Label")
+    ax = plt.title("Phenotypic Class Predicitions")
```

Comment:
Noticing that multiple visualizations may end up with the same title, would it make sense to provide an option for unique titles (building on what you have already)?

Reply:
I think at this point it doesn't make sense as these figures will be recreated with the data that is saved in intermediate files.